tests/unit/test_server_parse_status_metrics.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+from collections import namedtuple
+from prometheus_client.core import GaugeMetricFamily
+import xml.etree.ElementTree as ET
+
+from tableau_exporter.server_parser_status_metrics import TableauServerStatusParser
+
+class TestTableauServerStatusParser(unittest.TestCase):
+    DEFAULT_MACHINE_NAME='test'
+    DEFAULT_PROCESS_NAME='testprocess'
+    DEFAULT_STATUS='ActiveSyncing'
+    DEFAULT_XML='''<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+                <systeminfo xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+                    <machines>
+                        <machine name="{}">
```

Comment:
minor suggestion: using named arguments will be improve readability instead of positional arguments.